ashlar/scripts/preview_slide.py

```diff
@@ -30,6 +31,10 @@ def main(argv=sys.argv):
         "-c", "--channel", type=int, default=0,
         help="Channel number to display; default: 0",
     )
+    parser.add_argument(
+        "-s", "--scale", type=float, default=10,
+        help="Scaling factor used to downsample the image for display; default: 10",
+    )
```

Comment:
I see the emphasis of `Scaling factor used to downsample` but would it be more clear to just do `Scaling factor used` and set the default to 0.1 (and change the division to multiplication in the following changes)?

Reply:
What about "Downsampling factor for image display scaling"? I do think it's easier to consider and type in integer factors rather than their inverse.

Reply:
I see the ease of use by providing an integer factor than the inverse. In that case, maybe the key shouldn't be `--scale` but rather `--downsample-factor` (`-d` for short?)?